src/mock/task.js

```diff
@@ -0,0 +1,68 @@
+import {getRandomInteger} from "../utils.js";
+import {getRandomColor} from "../utils.js";
+
+const generateDescription = () => {
+  const descriptions = [
+    `Изучить теорию`,
+    `Сделать домашку`,
+    `Пройти интенсив на соточку`
+  ];
+
+  const randomIndex = getRandomInteger(0, descriptions.length - 1);
+
+  return descriptions[randomIndex];
+};
+
+const generateDate = () => {
+  const isDate = Boolean(getRandomInteger(0, 1));
+
+  if (!isDate) {
+    return null;
+  }
```

Comment:
Чем меньше код, тем меньше в нем багов :)
```suggestion
if (!getRandomInteger(0, 1)) {
    return null;
  }
```